src/main/java/com/labsynch/labseer/api/ApiStandardizationServicesController.java

```diff
@@ -144,6 +144,7 @@ public ResponseEntity<String> findDryRunDupeStructs() {
 
 	}
 
+	@Transactional
```

Comment:
I had to play a lot with transactional vs not when getting all this working.  I found that @Transactional here was not detrimental and made sense to have so I kept it.